tests/acceptance/features/lib/UserImpersonatePage.php

```diff
@@ -0,0 +1,89 @@
+<?php declare(strict_types=1);
+
+/**
+ * ownCloud
+ *
+ * @author Paurakh Sharma Humagain <paurakh@jankaritech.com>
```

Comment:
```suggestion
 * @author Paurakh Sharma Humagain <paurakh@jankaritech.com>
```
I was thinking you're swikriti ;)

Reply:
I was trying to impersonate Paurakh dai ;)